src/components/filterbox.css

```diff
@@ -0,0 +1,42 @@
+.filterbox {
```

Comment:
Some css files and some sass. Have we decided on sass? I don't really care, but Im more familiar with less.

Reply:
I'd like us to consider using postCSS (together with stylelint). It's conceptually similar to using babel and modern JavaScript.

Reply:
I'm putting my vote on postCSS

Reply:
Then go for that